pkg/cmd/util/clientcmd/factory.go

```diff
@@ -510,12 +510,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if t.Spec.Paused {
 				return true, nil
 			}
-			t.Spec.Paused = true
+			body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
somewhere in kube? I stoled this from the upstream PR https://github.com/kubernetes/kubernetes/pull/30919